docs/content/workers.md

```diff
@@ -24,6 +24,8 @@ In the GDKs [`Playground` project](../../workers/unity/Assets/Playground) we pro
 
 * `GameLogicScene`: This Scene contains only the `UnityGameLogic` prefab and can be used to build your server worker for cloud deployments.  
 
+The position of these prefabs in the Scene define the origin of the worker, which is used to translate all entities that have been added to this worker.
```

Comment:
Do we want a quick one-liner explanation of why we do this?

Reply:
sure, will add an one-liner to the one-liner